_mobile/ios.md

```diff
@@ -192,7 +191,38 @@ In the build settings, search for **other linker flags**.  Add a custom linker f
 
 ## API Docs
 
-Currently, the iOS framework uses the Pytorch C++ front-end APIs directly. The C++ document can be found here https://pytorch.org/cppdocs/. To learn more about it, we recommend exploring the [C++ front-end tutorials](https://pytorch.org/tutorials/advanced/cpp_frontend.html) on PyTorch webpage. In the meantime, we're working on providing the Swift/Objective-C API wrappers to PyTorch.
+Currently, the iOS framework uses the Pytorch C++ front-end APIs directly. The C++ document can be found here [https://pytorch.org/cppdocs/](https://pytorch.org/cppdocs/). To learn more about it, we recommend exploring the [C++ front-end tutorials](https://pytorch.org/tutorials/advanced/cpp_frontend.html) on PyTorch webpage. In the meantime, we're working on providing the Swift/Objective-C API wrappers to PyTorch.
```

Comment:
It's probably better to link to the cpp docs with a link that doesn't look like the address:
![image](https://user-images.githubusercontent.com/5652049/71199844-d2630480-2264-11ea-9a85-a7bee90774cb.png)